src/main/java/htsjdk/samtools/AbstractBAMFileIndex.java

```diff
@@ -398,14 +398,23 @@ protected void readChunks(int nChunks, List<Chunk> chunks) {
         }
     }
 
-    private void skipToSequence(final int sequenceIndex) {
+    protected void skipToSequence(final int sequenceIndex) {
     	//Use sequence position cache if available
     	if(sequenceIndexes[sequenceIndex] != -1){
     		seek(sequenceIndexes[sequenceIndex]);
     		return;
     	}
+
+        // Use previous sequence position if in cache
```

Comment:
Could you add an extra line pointing out how this helps in the common case of querying in sequential order.

Reply:
Done